Javascript/buystuff.js

```diff
@@ -1,394 +1,485 @@
-function buyAccelerator(autobuyer){
-	var autobuyamount = 0;
-	if(autobuyer){autobuyamount = 500}
-	var buythisamount = Math.max(player.coinbuyamount, autobuyamount)
-	while(player.coins.greaterThanOrEqualTo(player.acceleratorCost) && ticker < buythisamount) {
-
-		player.coins = player.coins.sub(player.acceleratorCost);
-		player.acceleratorCost = player.acceleratorCost.times(4);
-		player.acceleratorCost = player.acceleratorCost.dividedBy(costDivisor);
-		player.acceleratorBought += 1;
-	
-		if (player.acceleratorBought > (125 + 5 * player.challengecompletions.four)){
-		player.acceleratorCost = player.acceleratorCost.times(4 * player.acceleratorBought - (500 + 20 * player.challengecompletions.four))
-		}
-		if (player.acceleratorBought > (2000 + 5 * player.challengecompletions.four)){
-		player.acceleratorCost = player.acceleratorCost.times(Decimal.pow(2, player.acceleratorBought - (2000 + 5 * player.challengecompletions.four)))
-		}
-
-		if (player.currentChallenge == "four") {
-		player.acceleratorCost = player.acceleratorCost.times(Decimal.pow(10, player.acceleratorBought))
-        }
-        if (player.currentChallengeRein == "eight"){
-        player.acceleratorCost = player.acceleratorCost.times(Decimal.pow(1e50, player.acceleratorBought))
-        }
+function buyAccelerator(autobuyer) {
+  var autobuyamount = 0;
+  if (autobuyer) {
+    autobuyamount = 500
+  }
+  var buythisamount = Math.max(player.coinbuyamount, autobuyamount)
+  while (player.coins.greaterThanOrEqualTo(player.acceleratorCost) && ticker < buythisamount) {
+
+    player.coins = player.coins.sub(player.acceleratorCost);
+    player.acceleratorCost = player.acceleratorCost.times(4);
+    player.acceleratorCost = player.acceleratorCost.dividedBy(costDivisor);
+    player.acceleratorBought += 1;
+
+    if (player.acceleratorBought > (125 + 5 * player.challengecompletions.four)) {
+      player.acceleratorCost = player.acceleratorCost.times(4 * player.acceleratorBought - (500 + 20 * player.challengecompletions.four))
+    }
+    if (player.acceleratorBought > (2000 + 5 * player.challengecompletions.four)) {
+      player.acceleratorCost = player.acceleratorCost.times(Decimal.pow(2, player.acceleratorBought - (2000 + 5 * player.challengecompletions.four)))
+    }
+
+    if (player.currentChallenge == "four") {
+      player.acceleratorCost = player.acceleratorCost.times(Decimal.pow(10, player.acceleratorBought))
+    }
+    if (player.currentChallengeRein == "eight") {
+      player.acceleratorCost = player.acceleratorCost.times(Decimal.pow(1e50, player.acceleratorBought))
+    }
+
+    ticker++
+  }
+  ticker = 0;
+  player.prestigenoaccelerator = false;
+  player.transcendnoaccelerator = false;
+  player.reincarnatenoaccelerator = false;
+  updateAllTick();
+  if (player.acceleratorBought >= 5 && player.achievements[148] == 0) {

```

Comment:
These can probably still be one-liners.

Reply:
Redid that part due to your mention, i'd keep the if multiline, though, just find it making code easier to look at